src/test/java/org/fundacionjala/coding/danielcabero/movies/CustomerTest.java

```diff
@@ -0,0 +1,45 @@
+package org.fundacionjala.coding.danielcabero.movies;
+
+import java.util.List;
+import java.util.ArrayList;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Administrator on 7/5/2017.
+ */
+public class CustomerTest {
+
+    /**
+     * test 1.
+     */
+    @Test
+    public void testRentalEmpty() {
+        // given:
+        Customer customer = new Customer("Test");
```

Comment:
you are repeating this code in all your test
Consider put it in a @Before method